src/plugins/camera/camera_impl.cpp

```diff
@@ -345,6 +350,17 @@ MavlinkCommandSender::CommandLong CameraImpl::make_command_request_video_stream_
     return cmd_req_video_stream_info;
 }
 
+MavlinkCommandSender::CommandLong CameraImpl::make_command_request_video_stream_status()
+{
+    MavlinkCommandSender::CommandLong cmd_req_video_stream_status{};
+
+    cmd_req_video_stream_status.command = MAV_CMD_REQUEST_VIDEO_STREAM_STATUS;
```

Comment:
See: https://mavlink.io/en/messages/common.html#MAV_CMD_REQUEST_VIDEO_STREAM_STATUS

This command has been replaced by https://mavlink.io/en/messages/common.html#MAV_CMD_REQUEST_MESSAGE.

Reply:
Like this: https://github.com/mavlink/MAVSDK/blob/261c3516e72fe007fbc285f2903679bc27aadceb/src/plugins/camera/camera_impl.cpp#L291-L303

Reply:
Ok, we'll leave it as is for now and transition later.